packages/rich-text/src/component/index.js

```diff
@@ -137,6 +135,63 @@ function fixPlaceholderSelection( defaultView ) {
 	selection.collapseToStart();
 }
 
+function useReapply( {
```

Comment:
What's the purpose of splitting this out? What rule is it satisfying?

Reply:
Just want to group them all together so that it's clear that they are for the same logic.